src/components/ProductForm/ProductForm.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+const ProductForm = ({ id, url, colors }) => {
+  return (
+    <li className="productForm">
+      <div className="thumb">
+        <span className={id < 5 ? 'num' : 'num new'}>{`${id}`}</span>
```

Comment:
```suggestion
        <span className={id < 5 ? 'num' : 'num new'}>{id}</span>
```
클래스 조건을 좀 더 효율적으로 바꿔보면 어떨까요?
num은 공통이고, 조건에 따라 추가 클래스가 없거나 new 라는 클래스가 추가되거나의 경우네요!

Reply:
감사합니다 도현님 말씀하신 대로 바꾸어 보았습니다.